aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, Grid, Stack, Text } from 'grommet';
+import PropTypes from 'prop-types';
+
+// values chosen to best visually represent the margin area in diagram
+const largeColumns = ['13px', 'auto', '13px'];
+const smallColumns = ['8px', 'auto', '8px'];
+
+const MarginText = ({ label }) => (
+  <Stack anchor="bottom">
+    <Text alignSelf="center">{label}</Text>
+    {/* values chosen to best visually represent horizontal line in diagram. */}
+    <Box width="12px" height="1px" background="black" />
```

Comment:
Can we use theme edgeSize / borderSize for these values?

alternately, what if instead of stack we just did:
```
<Box border="bottom">
   <Text ... />
</Box>
```
would this work? just seeing if we can simplify the DOM

Reply:
good idea I think this would work